src/main/java/org/jabref/gui/groups/GroupDialog.java

```diff
@@ -466,7 +466,7 @@ private VBox createOptionsTexGroup() {
         texGroupBrowseButton.setOnAction((ActionEvent e) -> openBrowseDialog());
         texGroupHBox.getChildren().add(texGroupFilePath);
         texGroupHBox.getChildren().add(texGroupBrowseButton);
-        texGroupHBox.setHgrow(texGroupFilePath, Priority.ALWAYS);
+        HBox.setHgrow(texGroupFilePath, Priority.ALWAYS);
```

Comment:
What's this making for a difference?

Reply:
It's a static method, so there is no reason to call it from an instance variable (btw: automatic change by Intellj)